fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/bpe_tokenizer_trained_on_chembl_zinc_with_aug_4272372_samples_balanced_1_1.json

```diff
@@ -2783,6 +2783,51 @@
       "rstrip": false,
       "normalized": false,
       "special": true
+    },
+    {
+      "id": 309,
+      "content": "<AUTOENCODER_LATENT_LOG_VARIANCE>",
```

Comment:
What does each of the first 3 tokens do?

Reply:
hopefully this helps:
https://github.ibm.com/BiomedSciAI-Innersource/bmfm-bio/blob/autoencoder/bmfm_bio/generative/autoencoder/pl_data_module.py#L46

shortly:

```
"AUTOENCODER_TASK", #prefix for the task
"DECODED_FROM_LATENT",  #placeholder, not used yet
"AUTOENCODER_LATENT_LOG_VARIANCE", #log of the variance (a vector of values) - in an equivalent index in the model output, the model will output this
"AUTOENCODER_LATENT_MEAN", #mean of the variance (a vector of values) - in an equivalent index in the model output, the model will output this
"AUTOENCODER_LATENT_SAMPLED_Z", #both above are used to sample from the gaussian distribution, and it will be "injected" in this location
```

I believe all of this will be much clearer once I share the PRs for bmfm-core and bmfm-bio for autoencoder